samples/pipeline/README.md

```diff
@@ -0,0 +1,65 @@
+# Pipeline Sample
+
+This sample demonstrates how to submit a pipeline to Sagemaker for execution using your own JSON pipeline definition, using the AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.   
+
+## Prerequisites
+
+This sample assumes that you have completed the [common prerequisites](/samples/README.md).
+
+### Updating the Pipeline Specification
+
+In the `pipeline.yaml` file, modify the placeholder values with those associated with your account. 
+
+## Submitting your Pipeline Specification
+
+### Modify/Create a JSON pipeline definition
+
+Create the JSON pipeline definition using the JSON schema documented at https://aws-sagemaker-mlops.github.io/sagemaker-model-building-pipeline-definition-JSON-schema/. You will need to convert your JSON pipeline definition into String format to pass to the *.spec.pipelineDefinition* key in the Kubernetes YAML spec. You may use online third-party tools to convert from JSON to String format. In this sample, you are provided a sample pipeline definition with one Training step.
```

Comment:
It is possible to pass a json object as is using the [yaml-multi-line ](https://yaml-multiline.info/)directive. Something similar to https://github.com/aws-controllers-k8s/iam-controller/blob/main/test/e2e/resources/role_simple.yaml#L9-L21 . 

However not sure if the sagemaker API accepts indented json strings